eppo_core/src/ufc/eval_details.rs

```diff
@@ -0,0 +1,102 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+use crate::{AttributeValue, Attributes};
+
+use super::{AssignmentValue, Condition, FlagEvaluationError, Shard, Value};
+
+/// Details about feature flag evaluation.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EvalFlagDetails {
```

Comment:
Another field missing is `flagEvaluationDescription`. `allocations` provides more details than summary-like description, so not sure if we need it.

Reply:
The description is another nice-to-have for those who don't understand what "splits" mean by looking at the structure. It's also a generic error message that can be logged. The idea with this feature is to reduce support volume, and I don't think keeping this out helps that initiative.

Reply:
That said, this is just another "nice to have", and implementing it in every SDK might be more trouble than it's worth, so I'm fine leaving it out.

Reply:
Description is not added yet but I'll get to it later.